database/models/TransportModel.py

```diff
@@ -0,0 +1,20 @@
+from sqlalchemy import Column, Integer, String
+from database.db import Base
+
+
+class TransportModel(Base):
+    __tablename__ = 'transport'
+
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    model = Column(String)
+    year_of_production = Column(Integer)
+    mileage = Column(Integer)
+    body_type = Column(String)
+    color = Column(String)
+    price = Column(Integer)
+    orm_type = Column(String(50))
```

Comment:
make python dict TransportType like 
{MOTO: "MOTO"} and use it like  "polymorphic_identity":TransportType.MOTO ,

Reply:
1